Services/Interfaces/web_interface/static/js/components/config_tentacle.js

```diff
@@ -61,12 +61,25 @@ function handle_tentacle_config_update_error_callback(updated_data, update_url,
     create_alert("error", "Error when updating config", msg.responseText);
 }
 
+function check_config(){
+    var errors = configEditor.validate();
+
+    if(errors.length)
+        return false
+    else
+        return true
```

Comment:
```suggestion
    const errors = configEditor.validate();
    return !errors.length;
```
We prefer using const/let declaration in those js file to take advantage of modern js syntax

Reply:
👍